README.md

```diff
@@ -5,47 +5,32 @@
 [![Docker Repository on Quay](https://quay.io/repository/hmpps/hmpps-template-kotlin/status "Docker Repository on Quay")](https://quay.io/repository/hmpps/hmpps-template-kotlin)
 [![API docs](https://img.shields.io/badge/API_docs_-view-85EA2D.svg?logo=swagger)](https://hmpps-template-kotlin-dev.hmpps.service.justice.gov.uk/webjars/swagger-ui/index.html?configUrl=/v3/api-docs)
 
-This is a skeleton project from which to create new kotlin projects from.
+## Infrastructure Tech Stack
+Information on the technology stack chosen for the HMPPS-Integration-API project
 
-# Instructions
+### [Cloud Platform](https://user-guide.cloud-platform.service.justice.gov.uk/#cloud-platform-user-guide)
+A hosting platform which offers numerous tools such as logging, monitoring and alerting for our services.
```

Comment:
Might be worth adding that this is an MoJ thing that's built on top of AWS.

Reply:
I agree 100%!